src/main/cs/RLBotDotNet/RLBotDotNet/Renderer/Renderer.cs

```diff
@@ -91,6 +91,29 @@ public void DrawLine2D(Color color, Vector2 start, Vector2 end)
             _renderMessageOffsets.Add(finalOffset);
         }
 
+        /// <summary>
+        /// Draws a polyline between screen coordinates.
+        /// </summary>
+        /// <param name="color">The color of the line.</param>
+        /// <param name="vectors">The vectors of the line.</param>
+        public void DrawPolyLine2D(Color color, Vector2[] vectors)
+        {
+            if (vectors.Length < 2 || vectors.Length > 200) return;
```

Comment:
https://github.com/RLBot/RLBot/blob/8c96a4fb3218e4e39f127e83cd53cac02732692b/src/main/cs/RLBotDotNet/RLBotDotNet/Renderer/Renderer.cs#L99-L105

Function will now throw exception.
Removed maxVectors limit because it is addressed in other PR.